apps/web/src/serlo-editor-integration/create-plugins.tsx

```diff
@@ -82,7 +82,20 @@ export function createPlugins({
     },
     {
       type: EditorPluginType.Multimedia,
-      plugin: createMultimediaPlugin(),
+      plugin: createMultimediaPlugin({
+        allowedPlugins: [
+          EditorPluginType.Image,
+          EditorPluginType.Video,
+          ...(isProduction ? [] : [EditorPluginType.Audio]),
+          EditorPluginType.Geogebra,
+        ],
+        explanation: {
+          plugin: EditorPluginType.Rows,
+          config: {
+            allowedPlugins: [EditorPluginType.Text],
+          },
+        },
```

Comment:
note: without a migration we still have non text based plugins as explanations on serlo.org

Reply:
Oh right, good point. And if we want to go back, it would take a second migration...

Reply:
Good idea. Maybe long term, only allowing a text plugin makes sense. But for right now & since it is kind of an experiment, I would keep the rows plugin.